src/pipeline/parse.js

```diff
@@ -144,13 +138,12 @@ exports.getHighlightJS = (ast, paths) => {
     {}
   );
 
-  let js = `const rsh = require('${slash(path.join(rshPath, 'light'))}');`
-
+  let js = `var rsh = require('react-syntax-highlighter/dist/light');`
   Object.keys(languages).forEach((language) => {
     if (languageMap[language]) {
       language = languageMap[language];
     }
-    js += `\nrsh.registerLanguage('${language}', require('${slash(path.join(rshPath, 'languages', language))}').default);`
+    js += `\nrsh.registerLanguage('${language}', require('react-syntax-highlighter/dist/languages/${language}').default);`
   });
```

Comment:
@mathisonian I think `require('react-syntax-highlighter')` would fail any time the input project doesn't list it as a dependency.